docs_user/modules/proc_deploying-the-block-storage-services.adoc

```diff
@@ -2,38 +2,36 @@
 
 = Deploying the Block Storage services
```

Comment:
Where is this file being used?
I think we no longer have an import for it.

Reply:
I renamed the file. It is now called "Adopting the Block Storage service" and it is nested under the "Adopting Red Hat OpenStack Platform control plane services" assembly.

Reply:
But the file still exists, right? And it's not referenced anywhere.
I could be wrong, since the GH review interface is not all that great.

Reply:
I cannot download right now the PR to check, but I believe it should show as no content here if it was actually moved. I looks like to me that we have both files, and with different content.

Reply:
You're right. I misremembered what I did. I actually copied the content to a the `proc_adopting-the-block-storage-service.adoc` file and forgot to delete the `proc_deploying-the-block-storage-services.adoc`. I've now removed the `proc_deploying-the-block-storage-services.adoc` file.